packages/network/src/v2/common.ts

```diff
@@ -5,6 +5,7 @@ export const schemaTableId = new TableId("mudstore", "schema");
 export const metadataTableId = new TableId("mudstore", "StoreMetadata");
 
 export const storeEvents = ["StoreSetRecord", "StoreSetField", "StoreDeleteRecord"] as const;
+export const ephemeralEvents = ["StoreEphemeralSetRecord"] as const;
```

Comment:
Reminder to also change this if we change the event name to `StoreEphemeralRecord`